### ISSUE TITLE
cairo build works on SL5, fails on SL6

### ISSUE THREAD

**Issue by boegel** [easybuilders/easybuild-easyconfigs#639]
(parent issue: #638)

`goalf` and `goolf` builds fails with:

```
/usr/bin/ld: /path/to/software/GCC/4.7.2/lib64/libiberty.a(cplus-dem.o):
relocation R_X86_64_32S against `_sch_istable' can not be used when making a shared object; recompile with -fPIC
```

`ictce` builds fail with:

```
  CCLD   libcairo.la
.libs/cairo-ps-surface.o: In function `fread':
cairo-ps-surface.c:(.text+0x1df0): multiple definition of `fread'
.libs/cairo-png.o:cairo-png.c:(.text+0x850): first defined here
make[3]: *** [libcairo.la] Error 1
```

Very weird that this doesn't occur on SL5...

**Comment by boegel** [comment]
fixed with #1241

**Comment by Micket** [comment]
sounds like this was fixed 8 years ago. Good work boegel! :)
